src/sentry/processing/backpressure/topology.py

```diff
@@ -4,62 +4,19 @@
 In other words, which service (consumer) depends on which other services (queues, processing store).
 """
 
-from dataclasses import dataclass
-from typing import Mapping, Set, Union
-
-from django.conf import settings
-
-PROFILING_QUEUES = ["profiles.process"]
-PROCESSING_QUEUES = [
-    "events.preprocess_event",
-    "events.process_event",
-    "events.save_event",
-    "events.save_event_transaction",
-    "events.save_event_attachments",
-]
-SYMBOLICATION_QUEUES = [
-    "events.symbolicate_event",
-    "events.symbolicate_js_event",
-    "events.symbolicate_event_low_priority",
-    "events.symbolicate_js_event_low_priority",
-]
-REPROCESSING_QUEUES = [
-    "events.reprocess_events",
-    "events.reprocessing.preprocess_event",
-    "events.reprocessing.process_event",
-    "events.reprocessing.symbolicate_event",
-    "events.reprocessing.symbolicate_event_low_priority",
+PROCESSING_SERVICES = [
+    "celery",
+    "attachments-store",
```

Comment:
nit: The service name would technically be `attachments_cache` in code. 

Open question, since this is going to be spread across several repos: Should we try to align the names here with sentry services/utilities, or rather aim for consistency within?